tests/greenheart/test_hydrogen/test_eco.py

```diff
@@ -36,13 +37,13 @@ def setUpClass(self):
         filename_eco_config = os.path.join(orbit_library_path, f"plant/eco_config.yaml")
         filename_hopp_config = os.path.join(orbit_library_path, f"plant/hopp_config.yaml")
 
-        self.lcoe, self.lcoh, _ = run_simulation(filename_hopp_config, filename_eco_config, filename_turbine_config, filename_orbit_config, filename_floris_config, verbose=False, show_plots=False, save_plots=False,  use_profast=True, incentive_option=1, plant_design_scenario=1, output_level=4)
+        self.lcoe, self.lcoh, _ = run_simulation(filename_hopp_config, filename_eco_config, filename_turbine_config, filename_orbit_config, filename_floris_config, verbose=False, show_plots=False, save_plots=False,  use_profast=True, post_processing=True, incentive_option=1, plant_design_scenario=1, output_level=4)
 
     def test_lcoh(self):
-        assert self.lcoh == approx(5.70230272215567) # TODO base this test value on something
+        assert self.lcoh == approx(7.057994298481547) # TODO base this test value on something
```

Comment:
It looks like the change increased the LCOH and LCOE estimate considerably. They only change that should have had any impact was the loss percentage. I think it would be good to try setting the loss to 0.0 and just make sure the test results go back to the values before adding the default loss.

Reply:
The default loss was already being applied it was just a hardcoded value. The main difference in lcoh is coming from correcting the `nan` coming from ORBIT. FLORIS was interpreting those as turbines so for the test FLORIS was giving power for 12 turbines instead of 10.